easel/lib/screens/creator_hub/widgets/draft_list_tile.dart

```diff
@@ -108,74 +109,84 @@ class _DraftListTileState extends State<DraftListTile> {
     );
   }
 
+  void startPublishingFlowAgainPressed() {
+    widget.viewModel.saveNFT(nft: widget.nft);
+    Navigator.of(context).pushNamed(RouteUtil.kRouteHome);
+  }
+
   Widget getDraftCard() {
-    return DecoratedBox(
-      decoration: BoxDecoration(
-        color: Colors.white,
-        boxShadow: [
-          BoxShadow(
-            color: Colors.black.withOpacity(0.3),
-            offset: const Offset(0.0, 1.0),
-            blurRadius: 4.0,
-          ),
-        ],
-      ),
-      child: Padding(
-        padding: EdgeInsets.symmetric(horizontal: 15.w, vertical: 15.h),
-        child: Row(
-          children: [
-            buildAssetView(),
-            SizedBox(
-              width: 10.w,
+    return InkWell(
+      onTap: () {
+        widget.viewModel.startPublishingFlowAgain(startPublishingFlowAgainPressed: startPublishingFlowAgainPressed);
```

Comment:
Why not use this      
```
  onTap: () {
        startPublishingFlowAgainPressed();
      },
```

Reply:
We need to write test case for this that's why need to implement this function in viewModel. Modified `startPublishingFlowAgain` function